CRM/Core/Form.php

```diff
@@ -129,7 +129,7 @@ class CRM_Core_Form extends HTML_QuickForm_Page {
    *
    * @var int
    */
-  protected $_is_pay_later_enabled;
+  protected $isPayLaterEnabled;
```

Comment:
I notice that this variable is not used in this class. Is it used anywhere else? i.e. any risk of breaking something else?

Also I notice that most protected variables start with  `_`, but there are exceptions. Are there coding guidelines on this?

Reply:
@mlutfy it was an oldschool way of showing something was private https://stackoverflow.com/questions/663350/whats-the-deal-with-a-leading-underscore-in-php-class-methods

Reply:
Probably better to move any changes that are more clean up to a pr against master

Reply:
ie. this one could be used by any class that inherits from CRM_Core_Form so it might be better on master - the one below is fine as is as it is easy by reading it to feel confident about renaming that var